### ISSUE TITLE
Integration: Owntracks fails (INVALID_TAG)

### ISSUE THREAD

**Issue by danito** [home-assistant/frontend#10509]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Hi,
My system is in French.
When adding Owntracks integration, I get the message:
````
Success!
Translation error: INVALID_TAG
Configuration créée pour OwnTracks.
````

### Describe the behavior you expected

A response with a URL should be shown that one can enter in the configuration of OwnTracks

### Steps to reproduce the issue

HA in French.
Go to integration.
Add OwnTracks

### What version of Home Assistant Core has the issue?

core-2021.10.7

### In which browser are you experiencing the issue with?

FF, Edge

### Which operating system are you using to run this browser?

Win 10

**Comment by Raphencoder** [comment]
Same here !

**Comment by spacegaier** [comment]
Not actually a frontend issue, but instead an issue with the French translation of the the OwnTracks backend integration. I just corrected that, so with the next translation update (which I think only happens for major releases) those will the taken into account.

https://app.lokalise.com/project/130246255a974bd3b5e8a1.51616605/?view=single&reference_lang_id=640&single_lang_id=673&search=component::owntracks::config::create_entry::default

**Comment by spacegaier** [comment]
FYI: This fix has been manually included in the just published 2021.11.4 release

**Comment by mmerkenhof** [comment]
i have the same issue in dutch!
